plugins/services/src/js/reducers/serviceForm/FormReducers/Containers.ts

```diff
@@ -49,8 +59,12 @@ export function FormReducer(state, { type, path = [], value }) {
         this.cache.push({});
         break;
       case REMOVE_ITEM:
-        newState = newState.filter((item, index) => index !== value);
-        this.cache = this.cache.filter((item, index) => index !== value);
+        newState = newState.filter(
+          (_: any, itemIndex: number) => itemIndex !== value
```

Comment:
things we don't use are best declared `unknown` instead of `any`.

Reply:
✅